app/content-panel/tags/TagTable.tsx

```diff
@@ -24,7 +23,7 @@ const TagTable: React.FC = () => {
   };
 
   const handleDelete = (id: string) => {
-    console.log('Delete tag', id);
+    setTags(tags.filter((tag) => tag.id !== id));
```

Comment:
on delete we would call the method, and update the data in the table.
this is functionality for another ticket. remove it from here